imap-listener/README.md

```diff
@@ -0,0 +1,68 @@
+# IMAP Listener Service
+
+Long-running Node.js Service, der ein IMAP-Postfach überwacht und neue E-Mails an die Trassenscout-API weiterleitet.
+
+## Funktionalität
+
+- **Beim Start**: Alle nicht SEEN Mails werden abgearbeitet
+- **Im IDLE-Modus**: Neue eingehende Mails werden sofort verarbeitet
+- **Pro Mail**:
+  - Inhalt lesen
+  - API-Call an TS-API (Webhook)
+  - Bei Erfolg → Mail als SEEN markieren + Move zu `/DONE`
+  - Bei Fehler → Retry → Mail als SEEN markieren + Move zu `/ERROR`
```

Comment:
Diesen Teil sehe ich nicht im Script.
Ich würde das erstmal ganz einfach bauen und die Mail direkt in ERROR schieben ohne Retry.
Dann finden wir erstmal heraus, in welchen Fällen das eigentlich passiert. Vermutlich am ehesten, wenn der Webhook down ist während eines Deploys.